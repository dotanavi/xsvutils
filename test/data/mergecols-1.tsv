id	category	name	category
1	aa	record1	aa
2	bb;cc	record2	dd;ee
