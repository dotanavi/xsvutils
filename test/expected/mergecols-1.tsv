id	category	name
1	aa;aa	record1
2	bb;cc;dd;ee	record2
